### DISCUSSION TITLE
Lowcode Integration for Internal Tools/PoCs/MvPs

### DISCUSSION THREAD

**Discussion by pedrozadotdev** [pocketbase/pocketbase#4977] [Show and tell]
Guys, stopping by to ask for your feedback. I've been working on an integration between PocketBase and another tool (named PocketBlocks) for a few months now and I think it could add some value to the people here. In short, it is a low-code platform that facilitates the creation of PoCs/MVPs and internal tools. The integration is done between the OpenBlocks tool, which would be the frontend where you create applications using predefined components (drag and drop) or custom components made in react can also be used, and PocketBase as the backend. The entire platform is provided in a single binary as well. Several OpenBlocks features were made available for free (everything is actually free) such as branding, the possibility of embedding applications on other web pages, creating plugins, access control, authentication via email/username/Oauth providers(All PocketBase Auth Providers)... This integration arose from a personal need and I believe it can add value to other people (They just need to know that it exists 😅). Thank you in advance to anyone who can help (Star on GitHub/Upvote on PH/Suggestions).

[GitHub](https://github.com/internoapp/pocketblocks)
[Docs](https://pocketblocks.interno.app) 
[ProductHunt](https://www.producthunt.com/products/pocketblocks)

**Comment by SrGeneroso** [comment]
That sounds really cool! You got my star for sure, now I need to try it!!
Btw, it would be cool to have a video or some gifs in the docs and in the repo as you have in product hunt.
In any case, great idea, I think it could be really helpful and the combo with pocketbase is a great one.

**Reply by pedrozadotdev** [reply]
I started marketing the project this week. Improve the docs is one of the priorities. I'll take your suggestion. Thanks for the feedback.

**Comment by vrilcode** [comment]
Openblocks looks kind of dead to me. Last commit over a year ago.

**Reply by pedrozadotdev** [reply]
Yeah, it is. But the integration needs a bunch of changes in OpenBlocks codebase to work, so even if the project was alive a think I couldn't keep up the client updated with the main repo. I'm thinking to maintain the client(update dependences/fix bugs/add new features) and just change the server to sync with future PocketBase releases...

**Reply by yureckey** [reply]
> Yeah, it is. But the integration needs a bunch of changes in OpenBlocks codebase to work, so even if the project was alive a think I couldn't keep up the client updated with the main repo. I'm thinking to maintain the client(update dependences/fix bugs/add new features) and just change the server to sync with future PocketBase releases...

there is very actively developing "Lowcoder" service which is based on the OpenBlocks codebase

**Reply by pedrozadotdev** [reply]
That's true. Pocketblocks is based on the OpenBlocks codebase as well. In the beginning I thought to use the Lowcoder as a starting point but, as I said, Pocketblocks require a bunch of changes in the client, so I preferred to use the original codebase because Lowcoder was still an "early project" in that time. If somebody needs a full updated version of OpenBlocks, definitely look into Lowcoder, the team there is doing a great job.
